demo/config/locales/en.yml

```diff
@@ -1,3 +1,4 @@
 ---
 en:
-  hello: Hello world
+  errors:
+    format: '%{message}'
```

Comment:
@leeky Noticed that we've needed this here and in the existing apps using the form builder given the way we format error messages. Does that suggest we should be using `message` vs `full_message` in the library? Any objection to changing that here as we're moving this in?